hrui/hrui.php

```diff
@@ -651,3 +635,106 @@ function hrui_civicrm_alterContent( &$content, $context, $tplName, &$object ) {
   }
 }
 
+/**
+ * Add new information in the contact header as the contact photo,
+ * phone, department. All changes are made via Javascript.
+ *
+ * @param  [String] $content
+ * @return [String] Updated content markup
+ */
+function _hrui_updateContactSummaryUI($content) {
+  $contact_id = CRM_Utils_Request::retrieve( 'cid', 'Positive');
+  /* $currentContractDetails contain current contact data including
+   * Current ( Position = $currentContractDetails->position ) and
+   * ( Normal Place of work =  $currentContractDetails->location )
+  */
```

Comment:
so after this 

> one last minor note ... since this is going to be appended to the end of $content variable , there is no need to pass $content to _hrui_updateContactSummaryUI function , so you can change this line to :
`$content .= _hrui_updateContactSummaryUI($content);`

you can add this line here :
`$content = '';`